docs/dashboards/kubernetes-dashboard.asciidoc

```diff
@@ -0,0 +1,73 @@
+
+To check the daemonset’s status, run for example: `kubectl get pods -A`
+
+Once the daemonset is running, Elastic Endpoint will start sending Linux session data from Kubernetes to Kibana. You can then view that data using the Kubernetes dashboard.
+
+[discrete]
+== The Kubernetes dashboard
+To find the dashboard, go to **Security > Dashboards > Kubernetes**.
+
+image::kubernetes-dashboard.png[the Kubernetes dashboard, with numbered labels 1 through 3 for major sections]
+Sections numbered above are described below:
+
+  1. The charts at the top of the dashboard provide an overview of your monitored Kubernetes infrastructure. You can hide them by clicking Hide charts.
+  2. The Tree navigation menu allows you to navigate through your deployments and select the scope of the Sessions table to the right. You can select any item in the menu to show its sessions. In Logical view, the menu is organized by Cluster, Namespace, Pod, and Container image. In Infrastructure view, it is organized by Cluster, Node, Pod, and Container image.
+  3. The sessions table displays sessions collected from the selected element of your Kubernetes infrastructure. You can view it in fullscreen by selecting the button in its upper right corner, and sort it by any of its fields.
+
+You can filter the data using the KQL search bar and date picker at the top of the page, and take the following investigative actions on sessions in the sessions table using the buttons to their left:
+
+- View details
+- Open in {security-guide}/timelines-ui.html[timeline]
+- Run {kibana-ref}/osquery.html[Osquery]
+- {security-guide}/visual-event-analyzer.html[Analyze] event
+- Open {security-guide}/session-view.html[Session View]
+
+IMPORTANT: This dashboard uses data from the `logs-*` index pattern in the {security-guide}/advanced-settings.html#update-sec-indices[`securitySolution:defaultIndex` advanced setting] (where it is present by default). To collect data from multiple {es} clusters (as in a cross-cluster deployment), update `logs-*` to `*:logs-*` (this setting determines which data appears throughout Elastic Security).
+
+== Kubernetes-specific data in Session view
+
+Session view displays Kubernetes-specific metadata under the Metadata tab of the Detail panel:
+
+image::metadata-tab.png[the Detail panel's metadata tab]
+
+The metadata tab is organized into four expandable sections:
+
+- `Metadata` includes the fields `hostname`, `id`, `ip`, `mac`, and `name`, as well as information about the Host OS.
+- `Cloud` includes the fields `instance.name`, `provider`, `region`, `account.id`, and `project.id`.
+- `Container` includes the fields `id`, `name`, `image.name`, `image.tag`, and `image.hash.all`.
+- `Orchestrator` includes the fields `resource.ip`, `resource.name`, `resource.type`, `namespace`, `cluster.id`, `cluster.name`, and `parent.type`.
```

Comment:
It feels a little unnecessary to try to present this information in complete sentences, because it's not really readable like that. I wonder if this could be a table, or maybe a definition-list style bullet list, like:

> The **Metadata** tab is organized into expandable sections containing these fields:
> - **Metadata** — `hostname`, `id`, `ip`, `mac`, `name`, host OS information
> - **Cloud** — `instance.name`, `provider`, `region`, `account.id`, `project.id`
> - etc etc